.circleci/config.yml

```diff
@@ -30,17 +30,43 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
-      - run:
-          name: Run tests
-          command: |
-            BUILDKIT_PROGRESS=plain \
-            DOCKER_BUILDKIT=1 \
-            docker build \
-                --build-arg NODE_VERSION=<< parameters.node_version >> \
-                --build-arg PYTHON_VERSION=<< parameters.python_version >> \
-                -t snyk-python-plugin:integration-tests-<< parameters.python_version >> \
-                -f test/Dockerfile .
-            docker run --rm snyk-python-plugin:integration-tests-<< parameters.python_version >>
+      - when:
+          condition: 
+            equal: [ "3.12", <<parameters.python_version>>]
+          steps:
+          - run:
+              name: Run tests
+              command: |
+                BUILDKIT_PROGRESS=plain \
+                DOCKER_BUILDKIT=1 \
+                docker build \
+                    --build-arg NODE_VERSION=<< parameters.node_version >> \
+                    --build-arg PYTHON_VERSION=<< parameters.python_version >> \
+                    --build-arg PY_TEST_CMD=test:pysrc3_12 \
+                    -t snyk-python-plugin:integration-tests-<< parameters.python_version >> \
+                    -f test/Dockerfile .
+                docker run --rm snyk-python-plugin:integration-tests-<< parameters.python_version >>
+      - when:
+          condition: 
+            or:
+              - equal: [ "3.8", <<parameters.python_version>>]
```

Comment:
should not this include also `3.6` and `3.8`? range <3.6 to 3.11>?

Reply:
@gemaxim explained me why we are not introducing old versions (i.e. deprecated) resolving this one